ppr-ui/src/components/tables/RegistrationTable.vue

```diff
@@ -513,13 +513,18 @@ export default defineComponent({
       const draftItem = item as DraftResultIF
       const firstBaseReg = localState.registrationHistory[0].baseRegistrationNumber
       const firstDocId = localState.registrationHistory[0].documentId
-      if (item.baseRegistrationNumber && item.baseRegistrationNumber === firstBaseReg) {
-        return true
-      }
-      if (!item.baseRegistrationNumber && draftItem.documentId === firstDocId) {
-        return true
-      }
-      return false
+      return item.baseRegistrationNumber
+        ? item.baseRegistrationNumber === firstBaseReg
+        : draftItem.documentId === firstDocId
+    }
+
+    const isFirstItemMhr = (item: MhRegistrationSummaryIF | MhrDraftIF): boolean => {
+      const draftItem = item as MhrDraftIF
```

Comment:
We probably do not need to define `draftItem` and simply use `item` instead (in the condition below). Unless typescript is requiring.